Index: src/org/academiadecodigo/gnunas/sketch/GameObject/Collidable.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.academiadecodigo.gnunas.sketch.GameObject;\n\npublic interface Collidable {\n\n<<<<<<< HEAD\n=======\n    void collided(GameObject object);\n\n>>>>>>> CollisionDetector\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/org/academiadecodigo/gnunas/sketch/GameObject/Collidable.java	(revision ccb34fd5bbe7838842640ed39808e93f74c3cef9)
+++ src/org/academiadecodigo/gnunas/sketch/GameObject/Collidable.java	(date 1603038074378)
@@ -2,9 +2,5 @@
 
 public interface Collidable {
 
-<<<<<<< HEAD
-=======
     void collided(GameObject object);
-
->>>>>>> CollisionDetector
 }
